build.cmd

```diff
@@ -16,24 +16,20 @@ IF NOT EXIST %BUILD_FOLDER% (
     md %BUILD_FOLDER%
 )
 
-IF NOT EXIST %NUGET_PATH% (
-    IF NOT EXIST %CACHED_NUGET% (
-        echo Downloading latest version of NuGet.exe...
-        IF NOT EXIST %LocalAppData%\NuGet ( 
-            md %LocalAppData%\NuGet
-        )
-        @powershell -NoProfile -ExecutionPolicy unrestricted -Command "$ProgressPreference = 'SilentlyContinue'; Invoke-WebRequest 'https://dist.nuget.org/win-x86-commandline/%NUGET_VERSION%/nuget.exe' -OutFile '%CACHED_NUGET%'"
-    )
-
-    copy %CACHED_NUGET% %NUGET_PATH% > nul
+IF EXIST %NUGET_PATH% goto installkorebuild
```

Comment:
This is strictly semantics but I always had a problem with how we name the labels. The label should be "skipnugetinstall" or something like that and it should be linked to this block. Otherwise, if you change the order of the blocks, you have to update the labels.